### DISCUSSION TITLE
any way to look for keywords from both folder and parent? (eg. match `**/target/debug` only)

### DISCUSSION THREAD

**Discussion by ynejmidev** [sharkdp/fd#1648] [Q&A] [answered]
i tried a lot of stuff including:
- `fd -g '**/target/debug' --no-ignore` => no output
- `fd -g 'target' --and 'debug' --no-ignore` => no output
- `fd -s '^(target|debug)$' --no-ignore` => containing either target or debug (which i piped to grep and it works... last resort)

the fact that I dont really know how to articulate the problem coupled with how bad google has become, is getting me stuck on this, and i dont understand how something this simple is not..simple

**Comment by tavianator** [comment, marked as answer]
I think you just need to add --full-path

**Reply by ynejmidev** [reply]
Yes! thank you!
